CHANGELOG.md

```diff
@@ -1,26 +1,65 @@
+- Allow the admin to provide extra filter options in LDAP lookup (see [99daa00d565b](https://github.com/SUSE/Portus/commit/99daa00d565b))
+- Password length is no longer checked by Portus in LDAP (see [381fd61fb546](https://github.com/SUSE/Portus/commit/381fd61fb546))
+- Relaxed the requirements for user names, and removed the conflicts of user names in LDAP (see [a9d5a2646d0d](https://github.com/SUSE/Portus/commit/a9d5a2646d0d) and [215c681e65c2](https://github.com/SUSE/Portus/commit/215c681e65c2))
+- Introduce the `display_name` option (see [5d8c7e4bec97](https://github.com/SUSE/Portus/commit/5d8c7e4bec97))
+- Allow administrators to turn off smtp authentication (see [d837160bbe3e](https://github.com/SUSE/Portus/commit/d837160bbe3e))
+- Added an external hostname field to allow for events to come from other named services (see [0d58ed1fce0b](https://github.com/SUSE/Portus/commit/0d58ed1fce0b))
+- Added a help section to the menu (see [40a18a04b1fe](https://github.com/SUSE/Portus/commit/40a18a04b1fe))
+- Introduced more optional user restrictions (see [cddfb5924fae](https://github.com/SUSE/Portus/commit/cddfb5924fae))
+- Added the registry.catalog_page option (see [de4e4f4db74e](https://github.com/SUSE/Portus/commit/de4e4f4db74e))
+- Added option to disable change of visibility (see [50fb319ded81](https://github.com/SUSE/Portus/commit/50fb319ded81))
+- The signup form can now be disabled, and users can be created by the admin directly (see  [9bbd75cacd93](https://github.com/SUSE/Portus/commit/9bbd75cacd93) [f888460669d7](https://github.com/SUSE/Portus/commit/f888460669d7) [fa47c706a5db](https://github.com/SUSE/Portus/commit/fa47c706a5db)af](https://github.com/SUSE/Portus/commit/ [bbd75cacd935](https://github.com/SUSE/Portus/commit/bbd75cacd935) [888460669d77](https://github.com/SUSE/Portus/commit/888460669d77) [a47c706a5dba](https://github.com/SUSE/Portus/commit/a47c706a5dba)f),  [79bac5c4f54b](https://github.com/SUSE/Portus/commit/79bac5c4f54b) [58831c867fc0](https://github.com/SUSE/Portus/commit/58831c867fc0) [b0b567418cae](https://github.com/SUSE/Portus/commit/b0b567418cae)7d](https://github.com/SUSE/Portus/commit/ [9bac5c4f54b7](https://github.com/SUSE/Portus/commit/9bac5c4f54b7) [8831c867fc08](https://github.com/SUSE/Portus/commit/8831c867fc08) [0b567418cae7](https://github.com/SUSE/Portus/commit/0b567418cae7)d) and  [fcf20d7534e2](https://github.com/SUSE/Portus/commit/fcf20d7534e2) [1172713f82e0](https://github.com/SUSE/Portus/commit/1172713f82e0) [ef12abe14df0](https://github.com/SUSE/Portus/commit/ef12abe14df0)46](https://github.com/SUSE/Portus/commit/ [cf20d7534e2f](https://github.com/SUSE/Portus/commit/cf20d7534e2f) [172713f82e06](https://github.com/SUSE/Portus/commit/172713f82e06) [f12abe14df04](https://github.com/SUSE/Portus/commit/f12abe14df04)6))
```

Comment:
I can see that in multiple places... Moreover, some of the links are not found :disappointed: